package.json

```diff
@@ -16,7 +16,8 @@
   },
   "scripts": {
     "start": "node build-system/server/server-gae.js",
-    "start-local": "gulp",
+    "start-local": "env $(cat .env) gulp",
+    "start-remote": "env $(cat .env) node build-system/server/server-gae.js",
```

Comment:
Could you just update the `start` script instead of making a new "start-remote" script?